crates/client/src/client/schema/chain.rs

```diff
@@ -236,6 +237,8 @@ include_from_impls_and_cynic! {
         pub smo: DependentCost,
         pub srwq: DependentCost,
         pub swwq: DependentCost,
+
+        pub contract_root: U64,
```

Comment:
Have you tried to write a benchmark as a `DependentCost`? Maybe it is better if we charge users based on the size of the contract. But if there is not a huge difference in result, then it's okay to charge for possible max value.